server/app/routes/api/v1/va/ambassadors.js

```diff
@@ -204,14 +204,38 @@ async function updateTriplerLimit(req, res) {
     return error(404, res, "Ambassador not found")
   }
 
-  let newLimit = req.params.claim_tripler_limit;
+  let newLimit = req.params.claim_tripler_limit
   if (newLimit === null || newLimit === undefined) {
-    newLimit = "null";
+    newLimit = "null"
   }
 
-  req.neode.cypher("MATCH (a:Ambassador {id: $id}) SET a.claim_tripler_limit=toInteger($claim_tripler_limit)", {
+  req.neode.cypher(
+    "MATCH (a:Ambassador {id: $id}) SET a.claim_tripler_limit=toInteger($claim_tripler_limit)",
+    {
+      id: req.params.ambassadorId,
+      claim_tripler_limit: newLimit,
+    },
+  )
+
+  return _204(res)
+}
+
+/*
+ *
+ * updateAmbassadorAdminBonus(req, res)
+ * It gets its own function as some legacy ambassadors might not have this field, so we have to set it
+ *
+ */
+async function updateAmbassadorAdminBonus(req, res) {
+  let found = await req.neode.first("Ambassador", "id", req.params.ambassadorId)
+
+  if (!found) {
+    return error(404, res, "Ambassador not found")
+  }
+
+  req.neode.cypher("MATCH (a:Ambassador {id: $id}) SET a.admin_bonus=toInteger($admin_bonus)", {
```

Comment:
should this still be toInteger since the type of admin_bonus is number now?

Reply:
neo4j doesn't have "number" and I want to be sure it gets saved as something that looks like an integer since this is setting the property with cypher. It might be overkill though